cmsplugin_filer_file/templates/cmsplugin_filer_file/file.html

```diff
@@ -1,7 +1,9 @@
 {% if object.file.url %}
 <span class="file" {% if inherited_from_parent %} style="{{inherited_from_parent.style}}" {% endif %} >
 <a href="{{ object.file.url }}"{% if object.target_blank %} target="_blank"{% endif %}>
-{% if object.get_icon_url %}<img src="{{STATIC_URL}}{{ object.get_adjusted_icon_url }}" style="border: 0px" alt="Icon" />{% endif %}
+{% if object.get_icon_url %}
+    <img src="{% if object.is_absolute_url %}{{ object.get_icon_url }}{% else %}{{ PROXY_PREFIX }}{{ object.get_icon_url }}{% endif %}" style="border: 0px" alt="Icon" />
```

Comment:
Maybe you can remove the duplicated code ( object.get_icon_url ). Maybe something like this:
<code>{% if object.is_absolute_url == False %}{{ PROXY_PREFIX }}{% endif %}{{ object.get_icon_url }}</code>